dataset/dataset.py

```diff
@@ -78,7 +78,32 @@ def generate_datashop(context):
         results = process_part_attempts(partitioned_part_attempts[key], context)
         all_results.extend(results)
 
-    
+    tutor_keys = list_keys_from_inventory(section_ids, "tutor_message", source_bucket, inventory_bucket)
+    number_of_chunks = calculate_number_of_chunks(len(tutor_keys), chunk_size)
```

Comment:
It seems like `number_of_chunks` will be incorrect as it will only reflect the number of chunks of "tutor_message" and not the total number of chunks from "tutor_message" + "attempt_evaluated"

Reply:
Code updated to properly track number of chunks.